### DISCUSSION TITLE
Riftshadow Admin App

### DISCUSSION THREAD

**Discussion by Psypher9** [rezalas/riftshadow#218] [Ideas]
So, there are only so many tables, what kind of features are we hoping for in the Admin App? @sean-gilliam , @rezalas ?

**Comment by rezalas** [comment]
The big ones I'm wanting are all related to current playerbase and live server management.

- View active players
- Kick/ban active players on demand
- Reset player passwords on demand
- archive "dead" characters
- update configuration settings 
- start / restart game process 
- view logs

**Comment by sean-gilliam** [comment]
All I know is that @Daedalus11069 did an amazing job on the admin front-end.
